### PR TITLE
OCT-1945 Lower GP score giving the UQ score multiplier 1: client && OCT-1929 Change the UQ leverage to 0.01 & 1 client

### PR DESCRIPTION
**Pull request by aziolek**
## Definition of Done

1. [ ] If required, the desciption of your change is added to the [QA changelog](https://www.notion.so/octantapp/Changelog-for-the-QA-d96fa3b411cf488bb1d8d9a598d88281) 
2. [ ] Acceptance criteria are met.
3. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
4. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
5. [ ] Unit tests are added unless there is a reason to omit them.
6. [ ] Automated tests are added when required.
7. [ ] The code is merged.
8. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
9. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/src/components/Home/HomeGridRewardsEstimator/HomeGridRewardsEstimatorUqSelector/HomeGridRewardsEstimatorUqSelector.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const HomeGridRewardsEstimatorUqSelector: FC<HomeGridRewardsEstimatorUqSelectorP
   return (
     <div className={styles.root} data-test={dataTest}>
       <div className={styles.label}>{t('uq20+')}</div>
```
Comment (jmikolajczyk) [inline client/src/components/Home/HomeGridRewardsEstimator/HomeGridRewardsEstimatorUqSelector/HomeGridRewardsEstimatorUqSelector.tsx:21]: to remove

File: client/src/components/Allocation/AllocationLowUqScore/AllocationLowUqScore.tsx [hunk 2]
```diff
@@ -1,21 +1,78 @@
...
+      setIsRefetchRequired(false);
+    });
+  };
+
+  const windowOnBlur = () => {
+    setIsRefetchRequired(true);
+  };
+
+  useEffect(() => {
+    window.addEventListener('focus', windowOnFocus);
```
Comment (jmikolajczyk) [inline client/src/components/Allocation/AllocationLowUqScore/AllocationLowUqScore.tsx:50]: I suggest adding listeners only when user opens `GITCOIN_PASSPORT_CUSTOM_OCTANT_DASHBOARD` link.

**Comment by jmikolajczyk** [comment]
@aziolek we don't have waiting state when new value is fetching.

**Review by jmikolajczyk — CHANGES_REQUESTED** [review]

File: client/src/components/Home/HomeGridRewardsEstimator/HomeGridRewardsEstimatorUqSelector/HomeGridRewardsEstimatorUqSelector.tsx (hunk 1 above)
Comment (aziolek) [reply client/src/components/Home/HomeGridRewardsEstimator/HomeGridRewardsEstimatorUqSelector/HomeGridRewardsEstimatorUqSelector.tsx:21]: Done.

File: client/src/components/Allocation/AllocationLowUqScore/AllocationLowUqScore.tsx (hunk 2 above)
Comment (aziolek) [reply client/src/components/Allocation/AllocationLowUqScore/AllocationLowUqScore.tsx:50]: I thought about that but there is a slight change user may bump their score in another window, so maybe safer with focus?

**Review by jmikolajczyk — APPROVED** [review]

**Pull request merged**